Bio/File.py

```diff
@@ -82,16 +82,23 @@ def as_handle(handleish, mode='r', **kwargs):
     been deprecated (this happens automatically in text mode).
 
     """
-    try:
-        if sys.version_info[0] >= 3 and "U" in mode:
-            mode = mode.replace("U", "")
+    # Workaround for https://github.com/biopython/biopython/issues/1544.
+    # TODO: Remove when Python 3.5 dependency is dropped.
+    if sys.version_info[:2] < (3, 6):
```

Comment:
I think this needs to be also checking at least Python 3? Otherwise:

```python
ImportError: No module named pathlib
```

Reply:
Correction: At least Python 3.4
https://docs.python.org/3.4/library/pathlib.html

Specifically we don't want to do this on Python 2

Reply:
Ah, indeed ~ I missed the version 2 check. It's updated now, also with a more granular check for the minor version.